### PR TITLE
Reformat endpoints

### LINKED ISSUE #2: Resource URL names are not consistent
https://github.com/2DLenin/drfblog/blob/b7d4722acf83c3e9859bef61ef498afe7be3a568/littleblog/entry/urls.py#L5-L10

Some URLs are using CamelCase notation, some URLs are using snake_case notation in the naming. API is something that is usually exposed to the external world - integrations, other developers, etc. So it should be nice, logical, and consistent.

Also, it would be nice to add some nested structure where applicable. For example, you have endpoint to rate posts - it makes sense to have this endpoint nested under the posts endpoint. I mean smth like this: `/posts/<post id>/rate`. I used the naming I see more logical for this situation

### PR DESCRIPTION
**Pull request by Stanislav-Lykhachov**
Modified endpoints structure in entry app.

Fix #2

### CODE REVIEW

File: littleblog/entry/urls.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 from .views import UserProfileCreateView, EntryDetailView, RateUpdateView, EntryCreateView
 
 urlpatterns = [
-    path('CreateUserProfile/', UserProfileCreateView.as_view()),
-    path('EntryDetail/<int:pk>/', EntryDetailView.as_view()),
-    path('EntryCreate/', EntryCreateView.as_view()),
-    path('set_mark/', RateUpdateView.as_view())
+    path('create_profile/', UserProfileCreateView.as_view()),
+    path('entries/<int:pk>/', EntryDetailView.as_view()),
+    path('entries/create/', EntryCreateView.as_view()),
```
Comment (vlad-bezpalko) [inline littleblog/entry/urls.py:8]: By REST guidelines we should use http methods for separation of different actions on API resource.

There's some formal standart of RESTful API structure established. It's easier to explain on example, see this:

```

GET /users/  - list of all users;
POST /users/ - create user action;
GET /users/<pk>/  - specific user details;
PUT/PATCH /users/<pk>/ - update user details;
DELETE /users/<pk>/  - delete user. 

```
Follow that template in your applications and you won't be wrong. Adjust your URLs accordingly

**Review by vlad-bezpalko — CHANGES_REQUESTED** [review]
left comments above specifying required changes

File: littleblog/entry/urls.py (hunk 1 above)
Comment (Stanislav-Lykhachov) [reply littleblog/entry/urls.py:8]: So, based on this requirements, I need to combine each model's CRUD operations in one View? There is no another way to differ CRUD actions only based on http method with the same endpoints names? And than I have to merge and refactor my views to follow guideleness I guess.

Comment (vlad-bezpalko) [reply littleblog/entry/urls.py:8]: Not really. If you're using `View` approach you match one URL template with one `View` class. `/users/` and `/users/<pk>/` are different URLs and they should be connected to separate `View`s.

So, for listing all users and creating a new user you would use one View and for operations with specific user you would use another View. 

```
UsersListCreateView(generics.ListCreateAPIView):
    queryset ...
    serializer_class ...

    def perform_create():
        ....
```

Note that its possible to use different serializers for different actions. Example:
```
def get_serializer_class(self):
        if self.request.method == 'GET':
            return UserListSerializer
        elif self.request.method == 'POST':
            return UserCreateSerializer
```

Comment (vlad-bezpalko) [reply littleblog/entry/urls.py:8]: You can jump ahead and try to implement ViewSet. ViewSet is actually the thing that allows to write one class for one resource, covering all operation (CRUD + custom). https://www.django-rest-framework.org/api-guide/viewsets/

But I'd suggest to move to viewsets in further iterations / in separate PR.

**Pull request closed without merging**